src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,168 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    static final int DEFAULT_START_CAPACITY = 16;
+    static final int INCREASE_CAPACITY_VALUE = 2;
+    private int capacity;
+    private int size;
+    private int threshold;
+    private Node<K,V>[] nodes;
+
+    public MyHashMap() {
+        this.capacity = DEFAULT_START_CAPACITY;
+        this.size = 0;
+        this.threshold = (int) (DEFAULT_LOAD_FACTOR * capacity);
+        this.nodes = new Node[capacity];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= threshold) {
+            resizeList(key, value);
```

Comment:
If there is not enough place, just resize, and don't call ```return```, because you are adding new key value in the smae method. Don't duplicate logic.

Reply:
++, thank you, changed